driver-core/src/main/java/com/datastax/driver/core/TypeCodec.java

```diff
@@ -533,8 +535,8 @@ public DataType getCqlType() {
      * {@link TupleValue}, and the internal representation of a {@code ROWS} response),
      * which may appear in driver logs.</li>
      * </ol>
-     * If choose not to implement this method, you can return a constant string (for example
-     * "XxxCodec.format not implemented").
+     * If you choose not to implement this method, you should not throw an exception but
+     * instead return a constant string (for example "XxxCodec.format not implemented").
```

Comment:
Because we still do not catch it in the `QueryLogger`, for instance. Any exception raised in a `LatencyTracker` bubbles up to [here](https://github.com/datastax/java-driver/blob/3.0.x/driver-core/src/main/java/com/datastax/driver/core/Cluster.java#L1575) before being swallowed.
On a second thought, I think it would be even better to remove this paragraph altogether and protect our code against unchecked exceptions thrown by a codec, like we did in `BuiltStatement.toString()`.